web/src/components/nomination/AffiliationInput.vue

```diff
@@ -0,0 +1,58 @@
+<template>
```

Comment:
This is looking good. A further enhancement would be to DRY up this component with the job field, but we can save that for a future enhancement.